node/src/reactor.rs

```diff
@@ -191,6 +195,11 @@ pub trait Finalize: Sized {
     }
 }
 
+struct AllocatedMem {
```

Comment:
This struct needs documentation, badly. Especially since the units aren't in the member names.